src/renderer/src/assets/index.css

```diff
@@ -1,10 +1,8 @@
 body {
-  display: flex;
-  flex-direction: column;
   font-family: Roboto, -apple-system, BlinkMacSystemFont, 'Helvetica Neue', 'Segoe UI', 'Oxygen', 'Ubuntu',
     'Cantarell', 'Open Sans', sans-serif;
-  color: #86a5b1;
-  background-color: #2f3241;
+  color: #333333;
+  background-color: #f6f6f6;
```

Comment:
Define colors and fonts as variables, as they may change.

Reply:
yep, that said the theme/colour variables work is in a downstream branch so it's going to be much simpler if I fix it there.